dfencoder/autoencoder.py

```diff
@@ -1014,21 +1022,21 @@ def get_results(self, df, return_abs=False):
             pdf[ft] = df[ft]
             pdf[ft + '_pred'] = output_df[ft]
             pdf[ft + '_loss'] = mse[:, i].cpu().numpy()
-            pdf[ft + '_z_loss'] = mse_scaled[:, i].cpu().numpy()
+            pdf[ft + '_scaled_loss'] = mse_scaled[:, i].cpu().numpy()
 
         for i, ft in enumerate(self.binary_fts):
             pdf[ft] = df[ft]
             pdf[ft + '_pred'] = output_df[ft]
             pdf[ft + '_loss'] = bce[:, i].cpu().numpy()
-            pdf[ft + '_z_loss'] = bce_scaled[:, i].cpu().numpy()
+            pdf[ft + '_scaled_loss'] = bce_scaled[:, i].cpu().numpy()
 
         for i, ft in enumerate(self.categorical_fts):
             pdf[ft] = df[ft]
             pdf[ft + '_pred'] = output_df[ft]
             pdf[ft + '_loss'] = cce[:, i].cpu().numpy()
-            pdf[ft + '_z_loss'] = cce_scaled[:, i].cpu().numpy()
+            pdf[ft + '_scaled_loss'] = cce_scaled[:, i].cpu().numpy()
 
-        pdf['max_abs_z'] = combined_loss.max(dim=1)[0].cpu().numpy()
-        pdf['mean_abs_z'] = combined_loss.mean(dim=1).cpu().numpy()
+        pdf['max_abs_scaled_loss'] = combined_loss.max(dim=1)[0].cpu().numpy()
+        pdf['mean_abs_scaled_loss'] = combined_loss.mean(dim=1).cpu().numpy()
```

Comment:
Changing output column names will require updating code and docs in Morpheus. Need to create Morpheus PR for updates once this is merged.

Reply:
Change reverted